__tests__/components/search/QASearchResults.test.js

```diff
@@ -72,13 +72,21 @@ describe('<QASearchResults />', () => {
             selectable: false,
             drillable: false,
           },
+          {
+            property: 'Image URL',
+            values: [
+              'https://img.discogs.com/ilqScil5LIpcF_povstRcaEtEeg=/fit-in/600x527/filters:strip_icc():format(jpeg):mode_rgb():quality(90)/discogs-images/R-1622463-1425003720-8692.jpeg.jpg',
```

Comment:
Should this be fetched from the fixtures instead of discogs in the test?

Reply:
This is running in JSDOM, so images aren't actually rendered (or fetched). To verify, I ran this test after disconnecting from the network.